python/lsst/obs/subaru/strayLight/rotatorAngle.py

```diff
@@ -248,9 +249,14 @@ def inrStartEnd(header):
         Instrument rotator angle at end of exposure, degrees.
     """
 
-    inst_pa = header.getDouble('INST-PA')
-    ra_t_d = header.getDouble('CRVAL1')
-    de_t_d = header.getDouble('CRVAL2')
+    visitInfo = exposure.getInfo().getVisitInfo()
+    header = exposure.getMetadata()
+
+    inst_pa = 270.0 - visitInfo.getBoresightRotAngle().asAngularUnits(lsst.geom.degrees)
+    ra_t_sp, de_t_sp = visitInfo.getBoresightRaDec()
+
+    ra_t_d = ra_t_sp.asAngularUnits(lsst.geom.degrees)
+    de_t_d = de_t_sp.asAngularUnits(lsst.geom.degrees)
 
     mjd_str = header.getDouble('MJD-STR')
```

Comment:
Use `visitInfo.getDate()`.  This returns the middle of the exposure. Then use `visitInfo.getExposureTime()` to get the exposure time and add and subtract half of that. `ObservationInfo` does include explicit start and end times but obs packages do not yet have direct access to that.